src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor().name().toLowerCase();
+        int number = new Random().nextInt(100);
+        return new Ball(color, number);

```

Comment:
ColorSupplier and Random should be class fields

100 should be a constant
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)